plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -373,26 +373,28 @@ public void testDetectStaleness()
 
         assertUpdate("INSERT INTO base_table3 VALUES (3, DATE '2019-09-09'), (4, DATE '2019-09-10'), (5, DATE '2019-09-10')", 3);
         String plan = getExplainPlan("SELECT * from materialized_view_part_stale", ExplainType.Type.IO);
-        assertTrue(plan.contains("base_table3"));
+        assertThat(plan).contains("base_table3");
 
         plan = getExplainPlan("SELECT * from materialized_view_join_stale", ExplainType.Type.IO);
-        assertTrue(plan.contains("base_table3") || plan.contains("base_table4"));
+        Condition<String> containsTable3 = new Condition<>(p -> p.contains("base_table3"), "base_table3");
+        Condition<String> containsTable4 = new Condition<>(p -> p.contains("base_table4"), "base_table4");
+        assertThat(plan).is(anyOf(containsTable3, containsTable4));
```

Comment:
seems this remained.

Reply:
yeah we can't change this, because `assertThat(plan).contains("base_table3", "base_table4")` will be essentially an `and` condition as it expects both strings to be present, whereas the original condition was an `or`